npm_and_yarn/lib/dependabot/npm_and_yarn/dependency_files_filterer.rb

```diff
@@ -51,8 +51,8 @@ def package_required_lockfile?(lockfile)
         end
       end
 
-      def yarn_workspaces_lockfile?(lockfile)
-        return false unless lockfile.name == "yarn.lock"
+      def workspaces_lockfile?(lockfile)
+        return false unless ["yarn.lock", "package-lock.json"].include?(lockfile.name)
```

Comment:
Note: support npm 7 workspace projects

Reply:
To clarify, should we check to see if there are any workspaces defined in the package.json for npm7?

Reply:
Yep should already be covered by the line bellow 👇 `return false unless parsed_root_package_json["workspaces"]`